### ISSUE TITLE
 "Release Version: ... | Mifos X Release Date: ..." link at bottom of page is broken

### ISSUE THREAD

**Issue by AkashBhave** [openMF/community-app#2780]
## Description
The link at the bottom of all pages that states the _Release Version_ and _Release Date_ of the Mifos X web client is broken. When it is clicked, the user is redirected to a **404 error page**.

## Steps to Reproduce

1. Login to the Mifos X web client
2. Navigate to any page, or just stay on the homepage
3. Click on the link at the very bottom of the page that says "Release Version: ... | Mifos X Release Date: ..."

## Expected Behavior
This link should navigate to a page that shows a change log of the Mifos X web client, or another page that shows some sort of information about Mifos X. 

## Actual Behaviour
The link navigates to a **404 error page**.

## Settings
- Mifos X version: 17.07.01.RELEASE
- Mifos X release date: 14/Jul/2017
- Browser used: Google Chrome, Version 63.0.3239.84 (Official Build) (64-bit)

## Screenshots
![homepage](https://user-images.githubusercontent.com/16819026/34652216-568b0eca-f3a9-11e7-9b90-0ff2c5b848ce.PNG)

**Comment by mbj36** [comment]
[GitMate.io](https://gitmate.io) thinks the contributor most likely able to help you is @mbj36.

**Comment by tanmayanand44** [comment]
@mbj36 @AkashBhave Can I work on this issue?

**Comment by AkashBhave** [comment]
@tanmayanand44 Yes, you can work on this issue. I am not working on it at the moment.
